### PR TITLE
VoxelSniper: The journey through the Fabric of time. Part three: The ability to snipe a single block (and more)

### PR DESCRIPTION
**Pull request by KevinDaGame**
#217 

TODO:

- [x] Place a single block in the world
- [x] Implement the commands needed for the critial path for the snipe brush
- [x] /v command
- [x] Performers work (by accident
- [x] ball brush (and all shapes should) works

### CODE REVIEW

File: VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/blockstate/FabricBlockState.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        } catch (e: Exception) {
+            VoxelSniper.voxelsniper.getLogger().log(Level.WARNING, "Failed to set BlockData", e)
+            false
+        }
+
+    }
+
+    companion object {
+
+    }
```
Comment (Lennart99) [inline VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/blockstate/FabricBlockState.kt:49]: Why is this empty thing here?

File: VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/blockdata/FabricBlockData.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        return this.merge(blockData) == this
+    }
+
+    override fun getAsString(): String {
+        TODO("Not yet implemented")
+    }
+
+    override fun merge(newData: IBlockData): IBlockData {
+       TODO()
```
Comment (Lennart99) [inline VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/blockdata/FabricBlockData.kt:28]:
```suggestion
       TODO("Not yet implemented")
```

File: VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/chunk/FabricChunk.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    override fun getZ(): Int {
+        return chunk.pos.z
+    }
+
+    override fun getWorld(): IWorld {
+        return world
+    }
+
+    override fun getEntities(): MutableIterable<IEntity> {
+        throw NotImplementedError()
```
Comment (Lennart99) [inline VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/chunk/FabricChunk.kt:23]:
```suggestion
        TODO("Not yet implemented")
```

**Review by Lennart99 — APPROVED** [review]
LGTM, only a few small inconsistencies

File: VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/blockstate/FabricBlockState.kt (hunk 1 above)
Comment (KevinDaGame) [reply VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/voxelsniperfabric/blockstate/FabricBlockState.kt:49]: I think I implemented something in there and then didn't-. 
Companion object is where you put your statics btw

**Pull request merged**